### ISSUE TITLE
categorical_crossentropy instead of binary_crossentropy

### ISSUE THREAD

**Issue by kamua** [kusiwu/Resnet50-Cifar10-Python-Keras#1]
I think that **categorical_crossentropy** instead of **binary_crossentropy** should be used as loss function for multiple classes.

https://github.com/kusiwu/Resnet50-Cifar10-Python-Keras/blob/6f16cde98bb8b76dc65de5161c6e4986b27bcf58/train_resnet50_cifar10.py#L88

**Comment by kusiwu** [comment]
This was a shame for me. I have noticed this a few months ago but forgot to update this repo. Thanks you.
